bin/cron/deliver_poste_messages

```diff
@@ -293,7 +293,19 @@ def main
         raise
       end
 
-      POSTE_DB[:poste_deliveries].where(id: delivery[:id]).update(sent_at: sent_at)
+      is_dashboard_user = DASHBOARD_DB[:users].
+        where(hashed_email: hashed_email).
+        any?
+      is_dashboard_student = DASHBOARD_DB[:users].
+        where(hashed_email: hashed_email).
+        first[:user_type] == 'student' if is_dashboard_user
```

Comment:
Instead of querying the user twice, why not something like
```ruby
dashboard_user = DASHBOARD_DB[:users].
  where(hashed_email: hashed_email).
  first

is_dashboard_student = dashboard_user.nil? ? false : dashboard_user[:user_type] == 'student'

if is_dashboard_student
  # ...
```

Add a comment explaining why we're clearing the email here for students (i.e. why we needed it there temporarily there in the first place).